package.json

```diff
@@ -61,7 +62,8 @@
     "build:docs": "cd docs && sphinx-versioning -g ../ -l ./conf.py build docs ../docs/_build/html",
     "deploy:docs": "cd docs && sphinx-versioning -g ../ -l ./conf.py push docs gh-pages .",
     "rebuild": "rm -rf ./node_modules && npm install && ./node_modules/.bin/webpack",
-    "prepublishOnly": "npm run lint && npm run build:prod"
+    "prepublishOnly": "npm run lint && npm run build:prod",
+    "compile-typings": "tsc src/**/*.d.ts"
```

Comment:
I tried this locally on windows and it din't work. I have `typescript 2.7.2` in my global npm. Instead, `tsc src/index.d.ts` works for me.

Reply:
hmm weird i assume should work same on windows. here is on my ubuntu.
wmira@wmira-All-Series:/mnt/data/dev/pullreq/neon-js$ npm run compile-typings
> @cityofzion/neon-js@3.4.4 compile-typings /mnt/data/dev/pullreq/neon-js
> tsc src/**/*.d.ts

wmira@wmira-All-Series:/mnt/data/dev/pullreq/neon-js$ 

will get on the windows box and check why

Reply:
any updates?

Reply:
just tried it on windows now. Yeah weird that it is throwing error. Trying to see how to resolve